api/ApiHelper.tsx

```diff
@@ -942,17 +951,33 @@ export function initAPI(returnSSRResponse: boolean = false): API {
 
     let setRef = (refId: string): Promise<void> => {
         return new Promise((resolve, reject) => {
-            websocketHelper.sendRequest({
-                type: RequestType.SET_REF,
-                data: refId,
-                resolve: () => {
-                    resolve()
+            let googleId = localStorage.getItem('googleId')
+            if (!googleId) {
+                toast.error('You need to be logged in to use the ref system something.')
```

Comment:
I don't think showing an error to new users is a good idea.

Reply:
This should really never occur as the function is called in the onAfterLogin.
But if for some reason it is called before, why should I not send out an error? What should I do instead?